packages/@aws-cdk/aws-scheduler-targets-alpha/test/target.test.ts

```diff
@@ -0,0 +1,515 @@
+    const importedRole = Role.fromRoleArn(stack, 'ImportedRole', 'arn:aws:iam::123456789012:role/someRole');
+
+    const lambdaTarget = new LambdaInvoke(func, {
+      role: importedRole,
+    });
+
+    new Schedule(stack, 'MyScheduleDummy', {
+      schedule: expr,
+      target: lambdaTarget,
+    });
+
+    Template.fromStack(stack).resourceCountIs('AWS::Lambda::Permission', 0);
+    Template.fromStack(stack).hasResource('AWS::Scheduler::Schedule', {
+      Properties: {
+        Target: {
+          Arn: {
+            'Fn::GetAtt': ['MyLambdaCCE802FB', 'Arn'],
+          },
+          RoleArn: 'arn:aws:iam::123456789012:role/someRole',
+          RetryPolicy: {},
+        },
+      },
+    });
+
+    Template.fromStack(stack).hasResourceProperties('AWS::IAM::Policy', {
+      PolicyDocument: {
+        Statement: [
+          {
+            Action: 'lambda:InvokeFunction',
+            Effect: 'Allow',
+            Resource: [{
+              'Fn::GetAtt': ['MyLambdaCCE802FB', 'Arn'],
+            },
+            {
+              'Fn::Join': [
+                '', [
+                  { 'Fn::GetAtt': ['MyLambdaCCE802FB', 'Arn'] },
+                  ':*',
+                ],
+              ],
+            }],
+          },
+        ],
+      },
+      Roles: ['someRole'],
+    });
+  });
+
+  test('creates IAM policy for imported lambda function with imported IAM role in the same account', () => {
+    const importedFunc = lambda.Function.fromFunctionArn(stack, 'ImportedFunction', 'arn:aws:lambda:us-east-1:123456789012:function/somefunc');
+    const importedRole = Role.fromRoleArn(stack, 'ImportedRole', 'arn:aws:iam::123456789012:role/someRole');
+
+    const lambdaTarget = new LambdaInvoke(importedFunc, {
+      role: importedRole,
+    });
+
+    new Schedule(stack, 'MyScheduleDummy', {
+      schedule: expr,
+      target: lambdaTarget,
+    });
+
+    Template.fromStack(stack).resourceCountIs('AWS::Lambda::Permission', 0);
+    Template.fromStack(stack).hasResource('AWS::Scheduler::Schedule', {
+      Properties: {
+        Target: {
+          Arn: 'arn:aws:lambda:us-east-1:123456789012:function/somefunc',
+          RoleArn: 'arn:aws:iam::123456789012:role/someRole',
+          RetryPolicy: {},
+        },
+      },
+    });
+
+    Template.fromStack(stack).hasResourceProperties('AWS::IAM::Policy', {
+      PolicyDocument: {
+        Statement: [
+          {
+            Action: 'lambda:InvokeFunction',
+            Effect: 'Allow',
+            Resource: [
+              'arn:aws:lambda:us-east-1:123456789012:function/somefunc',
+              'arn:aws:lambda:us-east-1:123456789012:function/somefunc:*',
+            ],
+          },
+        ],
+      },
+      Roles: ['someRole'],
+    });
+  });
+
+  test('throws when lambda function is imported from different account', () => {
+    const importedFunc = lambda.Function.fromFunctionArn(stack, 'ImportedFunction', 'arn:aws:lambda:us-east-1:111122223333:function/somefunc');
```

Comment:
TIL as well, but take a look at this file [here](https://github.com/aws/aws-cdk/blob/apigateway-sagemaker-integration/.gitallowed)

tl;dr: cross env tests need to use `234567890123`